aggregators/1.0/modules/ROOT/pages/aggregators-studio-configuration.adoc

```diff
@@ -0,0 +1,148 @@
+= Use Studio to Configure Aggregators Module 1.0 - Mule 4
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a module in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the module to your Mule project>>.
+. <<configure-source,Configure a source for the module's flow>>.
+. <<add-connector-operation,Add an Aggregator Scope to the flow>>.
+. <<configure-os-aggregator,Configure an Object Store for an Aggregator>>.
+
+When you run the connector, you can view the app log to check for problems in real time, as described in <<view-app-log,View the App Log>>.
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the <link to the connector Reference Guide [<connector name> Reference].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+
+[[add-connector-to-project]]
+== Add the Module to Your Mule Project
+
+Add Aggregators Module to your Mule project to automatically populate the XML code with the module's namespace and schema location and to add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In the *Add Dependencies to Project* window, type `aggregators module` in the search field.
+. Click *Aggregators Module* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a module to a Mule project in Studio does not make that module available to other projects in your Studio workspace.
+
+
+[[configure-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these sources to use with Aggregators Module:
+
+* *HTTP > Listener*
++
+Initiates a flow each time it receives a request on the configured host and port.
+* *Aggregator listener*
++
+Initiates a sub-flow by listening to elements triggered by a referenced aggregator scope. +
+After the aggregator scope referenced by the listener completes an aggregation, the *Aggregator listener* triggers the sub-flow with a list of all the aggregated elements.
```

Comment:
```suggestion
After the aggregator scope referenced by the listener completes an aggregation, *Aggregator listener* triggers the subflow with a list of all the aggregated elements.
```
Are you sure about using bold here? We're essentially saying that the label triggers the subflow....